config/m4/ib.m4

```diff
@@ -58,10 +58,10 @@ AC_ARG_WITH([cm],
 #
 # mlx5 bare-metal support
 #
-AC_ARG_WITH([mlx5-hw],
-            [AC_HELP_STRING([--with-mlx5-hw], [Compile with mlx5 bare-metal support])],
+AC_ARG_WITH([mlx5-dv],
+            [AC_HELP_STRING([--with-mlx5-dv], [Compile with mlx5 DV support])],
```

Comment:
DV is very cryptic name. Let's be more specific what this option provides

Reply:
@Artemy-Mellanox pls update help message to "Compile with mlx5 direct-verbs support"

Reply:
I would even expand on this. "Direct Verbs (DV) support provides additional acceleration capabilities that are not available in a regular mode." or something along this lines.